app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -44,54 +44,21 @@ public SearchResultHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(final SearchResultHolder holder, int position) {
-        Datum datum = datumList.get(position);
+        final Datum datum = datumList.get(position);
+        holder.previewImageView.setVisibility(View.GONE);
         if (datum != null) {
             holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
             holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
-            if (!TextUtils.isEmpty(datum.getLink())) {
-                LinkPreviewCallback linkPreviewCallback = new LinkPreviewCallback() {
```

Comment:
@amitiwary999 I think there is some confusion. This code is not performing search by itself but is used for showing a preview link for the link provided by RSS. It is just like other search assistants and it is not required to be removed.